streaming/src/main/scala/org/apache/spark/streaming/StreamingContext.scala

```diff
@@ -269,6 +270,39 @@ class StreamingContext private[streaming] (
     RDDOperationScope.withScope(sc, name, allowNesting = false, ignoreParent = false)(body)
   }
 
+  private[streaming] val recoverableAccuNameToId: mutable.Map[String, Long] = {
```

Comment:
Yes it will add the strong references, that's why I said we need to reset the `StreamingContext.recoverableAccuNameToId` in `stop / shutdown` method.